styx-scheduler-service/src/main/java/com/spotify/styx/monitoring/MetricsStats.java

```diff
@@ -86,6 +87,7 @@
   private final ConcurrentMap<String, Histogram> dockerOperationHistograms;
   private final ConcurrentMap<String, Meter> dockerOperationMeters;
   private final ConcurrentHashMap<WorkflowId, Gauge> activeStatesPerWorkflowGauges;
+  private final ConcurrentHashMap<RunState.State, Gauge> activeStatesPerRunStateGauges;
```

Comment:
This map is not needed as the set of states is always static and we set up gauges on startup